v2/cloudconfig.go

```diff
@@ -1,19 +1,18 @@
-package v_0_1_0
+package v2
 
 import (
 	"bytes"
 	"compress/gzip"
 	"encoding/base64"
 	"text/template"
 
-	"github.com/giantswarm/clustertpr"
-	"github.com/giantswarm/clustertpr/spec"
+	"github.com/giantswarm/apiextensions/pkg/apis/cluster/v1alpha1"
 )
 
 type Params struct {
-	Cluster   clustertpr.Spec
+	Cluster   v1alpha1.Cluster
```

Comment:
We use the types of our generated clients here. They do not differ in structure so we can simply keep the templates as they are.